homeworks/roman.buzukov_BuzRom/3-async/js/script.js

```diff
@@ -0,0 +1,76 @@
+const containerEl = document.querySelector('[data-container]');
+const listEl = document.querySelector('[data-list]');
+const templateEl = document.querySelector('[data-post-template]');
+const postsEl = [];
+let filteredEl = postsEl;
+const sortEl = document.querySelector('[data-sort]');
+const filterEl = document.querySelector('[data-filter]');
+
+const getRender = (array) => {
```

Comment:
You get nothing here. This function returns nothing(

 So I consider renaming it to a simple `renderPosts` name. Also, it's better to specify your argument, what does it mean. For example, when I see an array, I don't understand what a type it is.

Can we rename it to a more specific name, e.g., `posts`?